README.md

```diff
@@ -1,6 +1,7 @@
 # libappc
 
 [![Build Status](https://travis-ci.org/cdaylward/libappc.svg?branch=master)](https://travis-ci.org/cdaylward/libappc)
+[![Build Status](https://webapi.biicode.com/v1/badges/manu343726/manu343726/libappc/master)](https://www.biicode.com/manu343726/libappc) 
```

Comment:
If do you want to own the `libappc` block, yep. Create a `cdaylward` account, change all references to my account, and don't forget to change parent block (See note bellow).